kaldb/src/main/java/com/slack/kaldb/metadata/schema/ChunkSchema.java

```diff
@@ -0,0 +1,65 @@
+package com.slack.kaldb.metadata.schema;
+
+import com.google.common.base.Objects;
+import com.slack.kaldb.metadata.core.KaldbMetadata;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Map;
+
+/** Schema for a chunk. */
```

Comment:
This needs better class documentation around what the purpose of the metadata is for, who uses it, and why this class is needed.